### ISSUE TITLE
Please add "attachments[]" field to class Message

### ISSUE THREAD

**Issue by videokojot** [Inumedia/SlackAPI#101]
Class Message is missing attachments[] field.
For example when calling method [https://api.slack.com/methods/im.history](url) using WebClient like this and there is message with attachments:
```

            WebClient wc = new WebClient();
            wc.QueryString.Add("channel", "CHANNEL");
            wc.QueryString.Add("token", "YOUR_TOKEN");
            string result = wc.DownloadString("https://slack.com/api/im.history");
            var result = System.Text.RegularExpressions.Regex.Unescape(result);
```
you will recieve json with attachments[] field property.

But when using SlackClient there is no prop. like this on Message class...

Please add it,

Thank you very much.

**Comment by juliepromact** [comment]
@cincibuch did you work on this issue ?

**Comment by cincibuch** [comment]
i did not. i just did not used SlackAPI for this case.

**Comment by juliepromact** [comment]
ok

**Comment by gpailler** [comment]
Duplicate of #136 
Fixed in the scope of #142
